app/helpers/concerns/foreman_bootdisk/hosts_helper_ext.rb

```diff
@@ -65,7 +65,7 @@ def bootdisk_button_disabled(host)
 
     # need to wrap this one in a test for template proxy presence
     def display_bootdisk_for_subnet(host)
-      if (proxy = host.try(:subnet).try(:tftp)) && proxy.has_feature?('Templates')
+      if (proxy = host.try(:subnet).try(:tftp) || host.try(:subnet).try(:httpboot)) && proxy.has_feature?('Templates')
```

Comment:
Here it is actually being required in order to show the buttons, so the test should definitely be there.